project/Settings.scala

```diff
@@ -37,7 +37,7 @@ object Settings {
     "-language:existentials", // Existential types (besides wildcard types) can be written and inferred
     "-unchecked", // Enable additional warnings where generated code depends on assumptions.
     "-Xcheckinit", // Wrap field accessors to throw an exception on uninitialized access.
-    "-Wconf:cat=deprecation:ws,any:e", // Fail the compilation if there are any warnings, except for deprecation warnings.
+    "-Wconf:any:e,cat=deprecation:ws", // Fail the compilation if there are any warnings, except for deprecation warnings.
```

Comment:
See the "Breaking Changes" section of https://github.com/scala/scala/releases/tag/v2.13.15: the order in which `-Wconf` is parsed has changed. This change necessary to retain functionality.

Of note: I am not sure this setting is actually what we want. A few lines above we specify:
```
"-deprecation", // Emit warning and location for usages of deprecated APIs.
```
but here we configure deprecations to be `ws`, which stands for "warning summary" - the summary suppresses the info about where the actual deprecations are in the codebase and just gives a summary like "12 deprecations….". BUT, to keep scope of this PR down I am not changing it here.